app/team.go

```diff
@@ -1591,7 +1591,7 @@ func (a *App) ClearTeamMembersCache(teamID string) {
 	page := 0
 
 	for {
-		teamMembers, err := a.Srv().Store.Team().GetMembers(teamID, page, perPage, &model.ViewUsersRestrictions{}, nil)
+		teamMembers, err := a.Srv().Store.Team().GetMembers(teamID, page, perPage, &model.TeamMembersGetOptions{ViewRestrictions: &model.ViewUsersRestrictions{}})
```

Comment:
nit: should we just `nil` out the entire `TeamMembersGetOptions` parameter, since it's just defaults all the way down?

Reply:
Yep! We can do that. It didn't come to mind when I was writing the code but since it's just defaults, we can nil the entire `TeamMembersGetOptions`.